tests/TransactionBuilderTest.php

```diff
@@ -0,0 +1,220 @@
+<?php
+declare(strict_types=1);
+
+use Illuminate\Support\Facades\DB;
+
+class TransactionBuilderTest extends TestCase
+{
+    protected $insertData = ['name' => 'klinson', 'age' => 20, 'title' => 'admin'];
+    protected $originData = ['name' => 'users', 'age' => 20, 'title' => 'user'];
+    protected $connection = 'mongodb_repl';
+    protected $originConnection = 'mongodb';
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        /** change connection to seplset? because the transaction needs  */
```

Comment:
There are some typos in this comment. On a separate note, my suggestion to switching the 27017 standalone over to a single-node replica set would allow us to remove this logic.